Include/codecs.h

```diff
@@ -141,6 +141,18 @@ PyAPI_FUNC(PyObject *) PyCodec_StreamWriter(
    Return 0 on success, -1 on error */
 PyAPI_FUNC(int) PyCodec_RegisterError(const char *name, PyObject *error);
 
+#if !defined(Py_LIMITED_API) || Py_LIMITED_API+0 >= 0x030e0000
```

Comment:
Please don't add this function to the limited C API. Undo Misc/stable_abi.toml change as well.

I suggest to add it as a private function to the internal C API instead: Include/internal/pycore_codecs.h.

Reply:
Ah I'm sorry. I actually didn't know whether to make it public or not. It'll be easy to make it public if someone wants it but for now, let's indeed keep it private! (I'll amend the PR tomorrow).